### DISCUSSION TITLE
How to use selector in windicss.

### DISCUSSION THREAD

**Discussion by xby020** [windicss/windicss#585] [Q&A]
This is described in the document:
<html>
<body>

sibling | .${utility} + * { ... } | Targets first sibling node.
-- | -- | --

</body>
</html>

I am puzzled when using this property. So in Tailwind, I can do like this:
```html
<label>
  <input type="checkbox" class="peer sr-only">
  <span class="h-4 w-4 bg-gray-200 peer-checked:bg-yellow-500">
   Test
  </span>

</label>
```
Does anyone provide the same implementation using windiCSS?

**Comment by abdulqodir18** [comment]
`sibling:checked:bg-dark-400` this way worked for me )
